deployment/testing-v4-gcloud-self/scripts/cleanup

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Delegate cleanup of buckets to the appropriate service account
+
+set -euo pipefail
+cd "$(dirname "$0")/.."
+
+pushd compose/server-b
+docker-compose run --rm app bin/cleanup
+# keep the container around...
+
+pushd ../../compose/server-a
```

Comment:
Maybe slightly clearly to use popd here, as you do elsewhere? Also, what does the comment `# keep the container around...` mean?

```suggestion
docker-compose run --rm app bin/cleanup
popd
# keep the container around...

pushd compose/server-a
```